crates/rolldown/tests/rolldown/warnings/commonjs_variable_in_esm/main.js

```diff
@@ -1,5 +1,6 @@
-exports.test = 1000;
+exports.test.res = 1000;
 exports.a = 1000;
+module.test = {}
 module.exports = {}
```

Comment:
reuse previous testcase, you could see the snapshot, the first `assignmentTarget` is not marked as `use commonjs in esm`

Reply:
`module.test` is not considered as `use commonjs in esm` too.